mis_builder/models/aep.py

```diff
@@ -67,9 +67,11 @@ class AccountingExpressionProcessor(object):
                          r"(?P<accounts>_[a-zA-Z0-9]+|\[.*?\])"
                          r"(?P<domain>\[.*?\])?")
 
-    def __init__(self, company):
-        self.company = company
-        self.dp = company.currency_id.decimal_places
+    def __init__(self, companies, currency_id=None):
+        self.companies = companies
+        self.currency_id = currency_id and currency_id or \
+            self.companies.env.user.company_id.currency_id
```

Comment:
Ok, but we might also consider that user wants to see the amounts in his own currency by default. But then we would need the Monetary cell as if not set, the traget currency depends on user